server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMailboxesMethodTest.java

```diff
@@ -96,8 +166,262 @@ public void setMailboxesShouldCreateMailbox() throws Exception {
         .when()
             .post("/jmap")
         .then()
+            .log().ifValidationFails()
```

Comment:
Ok, so put it everywhere then.
I guess it will produces too many logs and we'll want to disable this log right after you change, don't you think ?